app/src/test/java/io/apicurio/registry/ConfluentClientTest.java

```diff
@@ -184,81 +233,4 @@ public void testDelete() throws Exception {
-        testConverter(
-            subject,
-            name,
-            false,
-            (client) -> {
-                try {
-                    Schema schema = new Schema.Parser().parse(String.format("{\"type\":\"record\",\"name\":\"%s\",\"fields\":[{\"name\":\"bar\",\"type\":\"string\"}],\"connect.name\":\"%s\"}", name, name));
-                    int id = client.register(subject + "-value", schema);
-                    client.reset();
-                    // can be async ...
-                    Schema retry = retry(() -> client.getById(id));
-                    Assertions.assertNotNull(retry);
-                } catch (Exception e) {
-                    throw new IllegalStateException(e);
-                }
-            },
-            (c, b) -> {
-            }
-        );
-    }
-
-    @Test
-    public void testConverter_AutoRegisterSchema() {
-        String name = "myr" + ThreadLocalRandom.current().nextInt(0, Integer.MAX_VALUE);
-        testConverter(
-            generateArtifactId(),
-            name,
-            true,
-            (c) -> {
-            },
-            (client, bytes) -> {
-                try {
-                    client.reset();
-                    Schema retry = retry(() -> {
-                        ByteBuffer buffer = ByteBuffer.wrap(bytes);
-                        buffer.get(); // magic-byte
-                        int id = buffer.getInt();
-                        return client.getById(id);
-                    });
-                    Assertions.assertNotNull(retry);
-                } catch (Exception e) {
-                    throw new IllegalStateException(e);
-                }
-            }
-        );
-    }
-
-    private void testConverter(String subject, String name, boolean autoRegister, Consumer<SchemaRegistryClient> pre, BiConsumer<SchemaRegistryClient, byte[]> post) {
-        SchemaRegistryClient client = buildClient();
-
-        pre.accept(client);
-
-        org.apache.kafka.connect.data.Schema cs =
-            org.apache.kafka.connect.data.SchemaBuilder.struct()
-                                                       .name(name).field("bar", org.apache.kafka.connect.data.Schema.STRING_SCHEMA);
-        Struct struct = new Struct(cs);
-        struct.put("bar", "somebar");
-
-        AvroConverter converter = new AvroConverter(client);
-        Map<String, Object> config = new HashMap<>();
-        config.put(AbstractKafkaAvroSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG, "dummy");
-        config.put(AbstractKafkaAvroSerDeConfig.AUTO_REGISTER_SCHEMAS, autoRegister);
-        converter.configure(config, false);
-
-        byte[] bytes = converter.fromConnectData(subject, cs, struct);
-
-        post.accept(client, bytes);
-
-        SchemaAndValue sav = converter.toConnectData(subject, bytes);
-        Struct ir = (Struct) sav.value();
-        Assertions.assertEquals("somebar", ir.get("bar").toString());
-    }
```

Comment:
Why is this all removed?

Reply:
And the converter classes too.  Why are they all gone?

Reply:
The converter has a comment from @alesj indicating that can be removed once confluent changes Avro version from `1.8.x` to `1.9.x` and that's what happened in the version `5.5.0.`

Reply:
OK great, thanks.